src/consensus/mod.rs

```diff
@@ -0,0 +1,21 @@
+// Rust Bitcoin Library
+// Written by
+//   The Rust Bitcoin developers
+//
+// To the extent possible under law, the author(s) have dedicated all
+// copyright and related and neighboring rights to this software to
+// the public domain worldwide. This software is distributed without
+// any warranty.
+//
+// You should have received a copy of the CC0 Public Domain Dedication
+// along with this software.
+// If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+//
+
+//! Consensus
+//!
+//! This module defines structures, functions, and traits which are needed to
+//! conform to Bitcoin consensus.
+//!
+
+pub mod params;
```

Comment:
Can you add a `pub use params::Params` here so that people can use this as `bitcoin::consensus::Params`?

Reply:
Can you change this commit to address this?

Reply:
Addressed in: cccbb057bc88462284430b91ea333ea13ecbf7b0